tests/Fixer/ArrayNotation/YieldFromArrayToYieldsFixerTest.php

```diff
@@ -117,6 +117,29 @@ public static function provideFixCases(): iterable
             }',
         ];
 
+        yield [
+            '<?php function f() {
+                 '.'
+                    // uno
+                    yield 1;
+                    // duo
+                    yield 2;
+                    // tres
+                    yield 3;
+                '.'
+            }',
+            '<?php function f() {
+                yield from [
+                    // uno
+                    1,
+                    // duo
```

Comment:
```suggestion
                    // dos
```
It's most importantest of all changes 😆